doc/user/layouts/partials/sidebar.html

```diff
@@ -64,3 +67,45 @@
     <svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"><title>Close</title><path fill="none" stroke="currentColor" stroke-linecap="round" stroke-linejoin="round" stroke-width="32" d="M368 368L144 144M368 144L144 368"/></svg>
   </button>
 </div>
+
+<script>
+  function getBaseUrlWithoutVersion(currentPath, versions) {
```

Comment:
QQ: This isn't used anymore, yes?

Reply:
Yes, but I've probably done something wrong, have to review this code again.

Reply:
Nope, seems fine without it now, will remove it.